scripts/release.sh

```
@@ -0,0 +1,67 @@
+  if [ ${#LOCALTONET} -gt 0 ]; then
+    set -x
+    screen -S "localtonet.pid" -dm localtonet authtoken $LOCALTONET
+    set +x
+  fi
+}
+
+function boot() {
+  CMD=$1
+
+  shift
+  $CMD $@
```

Shellcheck (0.10.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)